src/screens/Docs/Stable.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import { MarkdownTemplate } from '../../components/MarkdownTemplate';
+import Page from '../../components/Page';
+
+const name = 'Stable Branch';
+
+const desc =
+  'Want to use the latest features and enhancments learn how to connect your project to the grommet stable.';
+
+const children = ` 
+
+
+Grommet stable is a Github branch that is used as a release that can be installed with NPM. It gets updated every time the [CircleCi](https://circleci.com/gh/grommet) builds passes on the Grommet master branch. The stable branch is updated multiple times a day as Grommet is constantly being enhanced.
```

Comment:
I wonder if we should mention that unit and visual snapshot tests have to pass. The text seems to imply that just a "build" succeeds.